package.json

```diff
@@ -66,60 +66,60 @@
     "report-dir": "coverage-cypress"
   },
   "devDependencies": {
-    "@cypress/code-coverage": "^3.8.7",
-    "@cypress/webpack-preprocessor": "^5.5.0",
+    "@cypress/code-coverage": "^3.9.4",
+    "@cypress/webpack-preprocessor": "^5.7.0",
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
     "@svgr/webpack": "^5.5.0",
     "@testing-library/react-hooks": "^3.7.0",
     "@types/enzyme": "^3.10.8",
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/jest": "^26.0.19",
-    "@types/react": "^16.14.2",
-    "@types/react-dom": "^16.9.10",
-    "@typescript-eslint/eslint-plugin": "^4.9.1",
-    "@typescript-eslint/parser": "^4.9.1",
-    "autoprefixer": "^10.1.0",
+    "@types/jest": "^26.0.22",
+    "@types/react": "^16.14.5",
+    "@types/react-dom": "^16.9.12",
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
+    "autoprefixer": "^10.2.5",
     "babel-core": "^6.26.3",
     "babel-jest": "^26.6.3",
     "clean-webpack-plugin": "^3.0.0",
-    "copy-webpack-plugin": "^6.4.0",
-    "css-loader": "^5.0.1",
+    "copy-webpack-plugin": "^6.4.1",
+    "css-loader": "^5.2.1",
     "cypress": "^5.6.0",
```

Comment:
There aren't very many tests so we could probably go to the latest version of cypress.

Reply:
Good point. Done.